MultiplayerMod/Platform/Steam/Network/Messaging/Surrogates/CarePackageInstanceDataSurrogate.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace MultiplayerMod.Platform.Steam.Network.Messaging.Surrogates;
+
+public class CarePackageInstanceDataSurrogate : ISerializationSurrogate, ISurrogateType {
+
+    public Type Type => typeof(CarePackageContainer.CarePackageInstanceData);
+
+    public void GetObjectData(object obj, SerializationInfo info, StreamingContext context) {
+        var node = (CarePackageContainer.CarePackageInstanceData) obj;
```

Comment:
Hmm, why is it a `node`? I see a "care package container" and "instance data", are graphs involved somehow?

Reply:
Done